### DISCUSSION TITLE
Standalone formatter release?

### DISCUSSION THREAD

**Discussion by acdha** [redhat-developer/vscode-xml#968] [Ideas]
It would be really helpful for some of my projects if the new formatter was available in a CLI form which could be used in a [pre-commit](https://pre-commit.com/) hook to have CI ensure that formatting drift doesn't creep in via editing processes not involving VS Code.

**Comment by fbricon** [comment]
It's probably doable, feel free to open a PR on the https://github.com/eclipse/lemminx repo to add a Main class for managing CLI. Our team doesn't have the bandwidth to handle such things at the moment.
